python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -328,6 +331,55 @@ def from_serialized(
             backfill_start_time=utc_datetime_from_timestamp(backfill_start_timestamp),
         )
 
+    @classmethod
+    def from_partitions_by_assets(
```

Comment:
A docstring explaining the behavior here would be helpful, i.e.:

Accepts a list of asset partition range selections, used to determine the target partitions to backfill.

For targeted assets, if partitioned and no partition ranges are provided, targets all partitions.

Reply:
added a docstring in new commit